iopipe_cli/stack.py

```diff
@@ -72,7 +71,7 @@ def modify_cloudformation(template_body, function_arn, token):
         }
     }
     # context = DeepChainMap({}, updates, template_body)
-    context = combine_dict(template_body, updates)
+    context = combine_dict.combine_dict(template_body, updates)
```

Comment:
Is combine_dict.combine_dict intended?

Reply:
Yeah, I changed the import to be consistent:
https://github.com/iopipe/iopipe-cli/pull/30/files#diff-ffe4fb384672c20b4ed295beccb18d81R5